### ISSUE TITLE
BatchNorm2d causes the inconsistency of output between frameworks

### ISSUE THREAD

**Issue by Crystal-yyy** [keras-team/keras#16508]
System information.
Ubuntu 20.04.2 LTS (GNU/Linux 5.4.0-74-generic x86_64)
TensorFlow 2.0.0 installed by anaconda
gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
NVIDIA-SMI 460.80 Driver Version: 460.80 CUDA Version: 11.2

Describe the problem.
Operators with BatchNorm2d often cause inconsistency between frameworks. And for this inconsistency, only changing the input may cause its nature to change.

For example.
structure：
`[{'name': 'ReLU6', 'params': {}}, {'name': 'BatchNorm2d', 'params': {'num_features': 1, 'eps': 2.5745934595451317e-06, 'momentum': 0.33708481957135894, 'affine': False, 'track_running_stats': True}}, {'name': 'Flatten'}, {'name': 'Dense', 'params': {'in_features': 784, 'out_features': 10}}, {'name': 'Softmax'}]`
（Inconsistency: When testing the three frameworks, the inconsistency here refers to comparing the output of the BatchNorm2d layer with the other two frameworks and computing their absolute values.）
